umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md

```diff
@@ -194,7 +205,7 @@ UMH uses TimescaleDB for database needs.
 After logging in as the root user, open a `psql` session with this command:
```

Comment:
also put it again here "After logging in as the root user with `sudo su`" (and before each command that requires it)